portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,109 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="en-US">
   <head>
     <meta charset="UTF-8">
```

Comment:
I think this is supposed to be lower case? https://google.github.io/styleguide/htmlcssguide.html#Encoding https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
I think DOCTYPE is a special case here: https://google.github.io/styleguide/htmlcssguide.html#Document_Type

Reply:
*Oh I was referring to the "UTF-8", sorry for not being clearer!

Reply:
Oh I see! Sorry about that. I'll make the change